backend/api/parse_pdf.py

```
@@ -1938,10 +1938,391 @@ async def parse_pdf_super_fast(
+            session_id=session_id,
+            stage="upload",
+            progress=50,
+            message=f"Processing file: {main_filename}"
+        )
+        
+        # Start async processing task with file contents (not UploadFile objects)
+        import asyncio
+        asyncio.create_task(_process_pdf_with_progress_tracking(
+            main_file_content, main_filename, main_content_type,
+            context_file_contents, save_to_db, session_id, db
+        ))
```

Ruff (0.13.3)

Store a reference to the return value of `asyncio.create_task`

(RUF006)